<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonAnderson">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Anderson</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBruce">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Bruce</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCastiel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Castiel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
